### PR TITLE
[TECH-394] Improve cypress execute step

### PR DESCRIPTION
**Pull request by Jorg88**
branch: feature/TECH-394-update-cypress-logic-for-monorepo
----
📕 [TECH-394](https://vandebron.atlassian.net/browse/TECH-394) Integrate cypress suites into mpyl jorgpost@vandebron.nl 
Make sure that certain cypress suites get triggered for certain projects when specified in the project.yml.

🏗️ Build [13](https://jenkins.k8s-serv-backend.vdbinfra.nl/job/MPyL%20Pipeline%20-%20Test/job/PR-114/13/display/redirect) ✅ Successful, started by _Jorg Post_  
🚀  *enrichChargeSessionsJob*, *job*, *kong-sync*, *[nodeservice](https://nodeservice-114.test.nl/)*, *[sbtservice](https://sbtservice-114.test.nl/)*  

[TECH-394]: https://vandebron.atlassian.net/browse/TECH-394?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/mpyl/steps/postdeploy/cypress_test.py [hunk 1]
```diff
@@ -33,28 +36,70 @@ def execute(self, step_input: Input) -> Output:
...
-        return Output(success=True, message=f"Cypress tests for {step_input.project.name} passed",
-                      produced_artifact=None)
+        return Output(success=True, message=f"Cypress tests for project {step_input.project.name} passed",
+                      produced_artifact=input_to_artifact(artifact_type=ArtifactType.JUNIT_TESTS, step_input=step_input,
+                                                          spec={TEST_OUTPUT_PATH_KEY: volume_path,
+                                                                "cypress_results_url": cypress_results_url}))
+
+    @staticmethod
+    def _target_to_test_target(target: Target) -> str:
+        if target == Target.PULL_REQUEST_BASE:
```
Comment (koeves) [inline src/mpyl/steps/postdeploy/cypress_test.py:100]: Don't you need PULL_REQUEST here as well?

**Review by koeves — APPROVED** [review]
Looks good in general!
I think I would move some of the commands/filenames from the execute() to a config object, but I understand if you think it's better like this!

File: src/mpyl/steps/postdeploy/cypress_test.py (hunk 1 above)
Comment (Jorg88) [reply src/mpyl/steps/postdeploy/cypress_test.py:100]: It's the default return value below

Comment (koeves) [reply src/mpyl/steps/postdeploy/cypress_test.py:100]: indeed

**Comment by Jorg88** [comment]
> Looks good in general! I think I would move some of the commands/filenames from the execute() to a config object, but I understand if you think it's better like this!

We can have a talk about it, I'm open for suggestions! It's just that if something never changes, then why make it configurable? "Future proofing" your code for something that isn't used/required right now is considered a bad practice afaik.

**Pull request merged**